notebooks/02_model/sar_deep_dive.ipynb

```diff
@@ -70,9 +70,9 @@
     "\n",
```

Comment:
<p>mmm why are we getting this variability in the result? SAR is a deterministic algo, we should get the same results</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/617/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
this was driving me crazy, i checked the results against staging and master and they were both the same as what I'm getting now (but different than what the notebook had previously). After some digging I discovered two things
1. Including the title_col parameter perturbs the dataset due to a join of the title information
2. <i>someone</i> presumably after contemplating the Answer to the Ultimate Question of Life, the Universe, and Everything changed the default seed in python_random_split from 123 to 42.
Removing both the title and the using seed 123 I confirm I get the same results as before, however I'd prefer to just update them to the new values as they are here.

Reply:
oh man sorry, it was me, yeah I changed the seed, makes sense to have the values with 42